ov/profiles.go

```diff
@@ -496,6 +498,7 @@ func (c *OVClient) CreateProfileFromTemplate(name string, template ServerProfile
 	new_template.ConnectionSettings = ConnectionSettings{
 		Connections: template.ConnectionSettings.Connections,
 	}
+	cleanup(&new_template)
```

Comment:
I think you have similar method in terraform sdk. Then shall remove it from there as it wouldn't be required anymore.

Reply:
We have it for server profile in Terraform. not for template.

Reply:
@yuvirani  both are used for creating profile from template. [Reff](https://github.com/HewlettPackard/terraform-provider-oneview/blob/master/oneview/resource_server_profile.go#L2067)